config/config_test.go

```diff
@@ -701,7 +708,66 @@ func TestConfig(t *testing.T) {
 				AuthorizedValues: []any{},
 				Type:             reflect.String,
 				IsSlice:          false,
+				Required:         true,
 			})
 		})
 	})
 }
+
+func TestRequiredConfig(t *testing.T) {
+	defaultLoader.mu.Lock()
+	loader := loader{
+		defaults: make(object, len(defaultLoader.defaults)),
+	}
+	loadDefaults(defaultLoader.defaults, loader.defaults)
+	defaultLoader.mu.Unlock()
```

Comment:
```suggestion
	loader := loader{
		defaults: make(object),
	}
```
No need to rely on the default loader here. You are not using those entries.